content/transactions/checkcash.md

```diff
@@ -0,0 +1,39 @@
+## CheckCash
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/app/tx/impl/CashCheck.cpp "Source")
+
+_Requires the [Checks Amendment](reference-amendments.html#checks)._
+
+Attempts to redeem a Check object in the ledger to receive up to the amount authorized by the corresponding [CheckCreate transaction][]. Only the `Destination` address of a Check can cash it with a CheckCash transaction. Cashing a check this way is similar to executing a [Payment][] initiated by the destination.
+
+Since the funds for a check are not guaranteed, redeeming a Check can fail because the sender does not have a high enough balance or because there is not enough liquidity to deliver the funds. If this happens, the
+
+Example CheckCash:
+
+```json
+{
+    "Account": "rfkE1aSy9G8Upk4JssnwBxhEv5p4mn2KTy",
+    "TransactionType": "CheckCash",
+    "Amount": "100000000",
+    "CheckID": "838766BA2B995C00744175F69A1B11E32C3DBC40E64801A4056FCBD657F57334",
+    "Fee": "12"
+}
+```
+
+In addition to the [common fields](#common-fields), a CheckCancel transaction has the following:
+
+| Field       | JSON Type | [Internal Type][] | Description                    |
+|:------------|:----------|:------------------|:-------------------------------|
+| `CheckID`   | String    | Hash256           | The ID of the [Check ledger object](reference-ledger-format.html#check) to cash, as a 64-character hexadecimal string. |
+| `Amount`    | [Currency Amount][] | Amount  | _(Optional)_ Redeem the Check for exactly this amount, if possible. The currency must match the `SendMax` of the corresponding CheckCreate transaction. |
+| `DeliverMin` | [Currency Amount][] | Amount | _(Optional)_ Redeem the Check for at least this amount and as much as possible. The currency must match the `SendMax` of the corresponding CheckCreate transaction. |
```

Comment:
While you have a comment below I'd suggest calling out that one of these fields is required, perhaps because that's what we've done in some similar cases for xCurrent. I'd add a variation to both, something like this for `DeliverMin`:
> | `DeliverMin` | [Currency Amount][] | Amount | _(Optional)_ Redeem the Check for at least this amount and as much as possible. The currency must match the `SendMax` of the corresponding CheckCreate transaction. Either this field or `Amount` is required.